server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/GetMessageListMethodTest.java

```diff
@@ -240,6 +240,152 @@ public void getMessageListShouldFilterMessagesWhenMultipleInMailboxesFilterMatch
     }
 
```

Comment:
If you mean filter={} it still doesn't work as jackson can't tell what is the type of filter (FilterCondition or FilterOperator).
I'm not fixing it now as it will be more pertinent when handling filter operators.
Anyway can you always use filter = null (or no filter) which should be already tested